elements/pfe-sass/maps/_typography.scss

```diff
@@ -81,3 +81,148 @@ $pf-type-sizing--titles: (
   title--m-sm--FontSize:    #{pfe-var(FontSize--sm ) },   // 14px
   title--m-xs--FontSize:    #{pfe-var(FontSize--xs ) },   // 12px
 );
+
+$pf-typography-sets: (
+  title: (
+    "6xl": (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-6xl--FontSize), // alt: could use pfe-var(FontSize--6xl ) directly instead of abstracting it above
```

Comment:
We can keep the references in separate maps if you prefer or, if you don't think there's value in it, we can migrate them directly into this map too.